react_ujs/index.js

```diff
@@ -155,6 +163,15 @@ var ReactRailsUJS = {
     detectEvents(this)
   },
 
+  unmountRoot: function(node) {
+    var targetRoots = this.roots.filter(function(rootElement) {return rootElement["node"] && (rootElement["node"] === node)})
```

Comment:
we should have at least eslint -- this formatting is bad...

Reply:
I agree.
Once I tried to add eslint configuration, but the result was much worse than adding Rubocop. Then I stopped it.